frontend/src/components/common/select/select.tsx

```diff
@@ -1,41 +1,37 @@
+import SelectReact from 'react-select';
 import { ErrorMessage } from '@hookform/error-message';
-import {
-  Control,
-  FieldErrors,
-  useController,
-  Path,
-  FieldValues,
-} from 'react-hook-form';
-
+import { Control, FieldErrors, Path, FieldValues } from 'react-hook-form';
 import styles from './styles.module.scss';
 
 type Props = {
-  options: string[],
+  options: string[];
   placeholder?: string;
   label: string;
   name: Path<FieldValues>;
   control: Control;
   errors: FieldErrors;
 };
 
-const Select: React.FC<Props> = ({
-  options,
-  label,
-  name,
-  control,
-  errors,
-}) => {
-  const { field } = useController({ name, control });
+type Options = {
```

Comment:
```suggestion
type Option = {
```
move it to `frontend/common/types/ui`